e2eTests/test_task_journey.py

```diff
@@ -0,0 +1,93 @@
+from time import sleep
+from threading import Thread
+
+import vcr
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.wait import WebDriverWait
+from selenium.webdriver.support import expected_conditions
+
+import importlib
+import os
+import pytest
+
+from e2eTests.utils.temp_trello_boards import create_trello_board, delete_trello_board
+from todo_app import app
+from dotenv import find_dotenv, load_dotenv
+import todo_app.data.trello_client
+
+
+@pytest.fixture(scope='module')
+def app_with_temp_board():
+    # Create the new board & update the board id environment variable
+    with vcr.use_cassette(
+            "cassettes/e2etest_setup.yaml", filter_query_parameters=['key', 'token'], ignore_localhost=True
+    ):
+        board_id = create_trello_board("test board")
+        file_path = find_dotenv('.env')
+        load_dotenv(file_path, override=True)
+        importlib.reload(todo_app.data.trello_client)
+        original_board_id = os.environ.get('TRELLO_TO_DO_BOARD_ID')
+        todo_app.data.trello_client.TRELLO_TO_DO_BOARD_ID = board_id
```

Comment:
(In my opinion this behaviour might be better hidden if the TrelloClient handled loading these environment variables when needed rather than when the file is imported as per the last review, but certainly reasonable here)